sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/TextPollOptionView.kt

```diff
@@ -144,27 +144,37 @@ internal class TextOptionView(context: Context?) : RelativeLayout(context) {
     private fun initializeViewStyle(optionStyle: TextPollBlockOptionStyle) {
         val borderRadius = optionStyle.borderRadius.dpAsPx(resources.displayMetrics).toFloat()
         val borderStrokeWidth = optionStyle.borderWidth.dpAsPx(resources.displayMetrics).toFloat()
-        val doubleBorderStrokeWidth = borderStrokeWidth * 2
 
         val borderPaint = Paint().create(
             optionStyle.borderColor.asAndroidColor(),
             Paint.Style.STROKE,
-            doubleBorderStrokeWidth
+            borderStrokeWidth
         )
         val fillPaint = Paint().create(optionStyle.background.color.asAndroidColor(), Paint.Style.FILL)
         val resultPaint = Paint().create(optionStyle.resultFillColor.asAndroidColor(), Paint.Style.FILL)
 
+        borderView.borderPaint = borderPaint
+
         optionPaints = OptionPaints(borderPaint, fillPaint, resultPaint)
 
         val rect = RectF(0f, 0f, width.toFloat(), height.toFloat())
-        roundRect = RoundRect(rect, borderRadius, doubleBorderStrokeWidth)
+        roundRect = RoundRect(rect, borderRadius, borderStrokeWidth)
+        halfBorderWidth = (optionStyle.borderWidth / 2).dpAsPx(resources.displayMetrics).toFloat()
+        borderView.borderRoundRect = roundRect?.copy(rectF = RectF(halfBorderWidth, halfBorderWidth,
+            width.toFloat() - halfBorderWidth,
+            height.toFloat() - halfBorderWidth))
```

Comment:
Hm. Is it reasonable to have borderView accept a `Border` value directly, and do this bit of arithmetic itself?

there are arguments for and against, just wondering if that would help tidy things up a bit.

Reply:
I changed it to accept a `border`, did think it made the `TextPollOptionView` and `ImagePollOptionView` a little cleaner